antarest/core/serde/ini_reader.py

```diff
@@ -377,3 +377,17 @@ def read(self, path: Any, **kwargs: Any) -> JSON:
         sections = super().read(path)
         obj = cast(Mapping[str, JSON], sections)
         return obj[self._section_name]
+
+
+def read_ini(content: TextIO) -> JSON:
```

Comment:
Do you use this aside from the tests ?

Reply:
No! but it makes sense to have this feature in this module from my point of view, although it's not used for now in prod code.
I prefer this as opposed to have an additional module only for testing purposes.